go.mod

```diff
@@ -40,3 +40,32 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29 // resolve `case-insensitive import collision` for gnostic/openapiv2 package
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.6.1
 )
+
+// https://nvd.nist.gov/vuln/detail/CVE-2021-3121 (skippy peanut butter)
+// TODO: Remove replace once one of the dependencies that pulls in `protobuf`
+//       is updated to a level that includes the fixed version as part of
+//       their dependency tree:
+//       - github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher
+//       - github.com/Microsoft/hcsshim
+//       - github.com/containerd/cgroups
+//       - github.com/containerd/ttrpc
+//       - github.com/coreos/prometheus-operator
+//       - github.com/deislabs/oras
+//       - github.com/dhui/dktest
+//       - github.com/golang-migrate/migrate/v4
+//       - github.com/grpc-ecosystem/go-grpc-middleware
+//       - github.com/lightstep/lightstep-tracer-common/golang/gogo
+//       - github.com/lightstep/lightstep-tracer-go
+//       - github.com/prometheus/alertmanager
+//       - github.com/spf13/viper
+//       - github.com/thanos-io/thanos
+//       - go.etcd.io/etcd
+//       - k8s.io/api
+//       - k8s.io/apiextensions-apiserver
+//       - k8s.io/apimachinery
+//       - k8s.io/apiserver
+//       - k8s.io/client-go
+//       - k8s.io/code-generator
+//       - k8s.io/kube-state-metrics
+//       - k8s.io/metrics
+replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
```

Comment:
Doesn't adding `github.com/gogo/protobuf` as a required module fix this instead? This module is on v1.x, my understanding is that any import of protobuf will resolve to the declared version.

Reply:
If replace is the only way to address this, please put it in the "replace" block above.

Reply:
I can try that. Thanks for the hint. My rational was that I wanted this to stick out from the other dependencies due to its somewhat special nature, i.e. indirect dependency that needs to explicitly pinned to a specific version.